pkg/hubble/metrics/http/handler.go

```diff
@@ -196,3 +221,7 @@ func observerObserve(o prometheus.Observer, value float64, traceID string) {
 		o.Observe(value)
 	}
 }
+
+func (h *httpHandler) Deinit(registry *prometheus.Registry) bool {
+	return registry.Unregister(h.requests) || registry.Unregister(h.responses) || registry.Unregister(h.duration)
```

Comment:
An error in one unregistration means the others do not get unregistered. This doesn't seem desirable. Perhaps you should accumulate the errors instead.

Reply:
Fixed